python/minifi/__init__.py

```diff
@@ -66,9 +66,9 @@ def setBase(self, proc):
     def get(self, session):
         ff = self._minifi.get(self._instance.get_instance(),self._flow, session)
         if ff:
```

Comment:
Python also supports ternary:
```
return FlowFile(self._minifi, ff) if ff else None
```

Reply:
Yeah i tend to avoid ternary operators in general since my experience with production code has always led maintainers at some point to result in error. Looks great -- but more errors than positives in my experience I'm afraid.

Reply:
Fair point.